Bulkgate/Database.php

```diff
@@ -50,7 +52,7 @@ public function lastId()
 
     public function escape($string)
     {
-        return addslashes($string);
+        return str_replace(["'", '"'], ["\'", '\"'], $string);
```

Comment:
Myslíš, že tohle je bezpečný?

Reply:
ok, vymenim zpet za addslashes, menil jsem to kvuli coding snifferu, ale ted uz mi to nehlasi warning.

Reply:
No osobně bych použil nějakou metodu přímo magenta na escapování.

Reply:
V diskuzi jsem nasel, ze se da pouzit PDO::quote(), takze jsem to tam dal a jeste to musim poradne otestovat.

Reply:
jojo supr